mangaki/mangaki/tests/test_refresh_from_anidb.py

```diff
@@ -87,6 +87,6 @@ def test_refresh_tags_from_anidb_confirmed(self):
         }
 
         response = self.client.post(refresh_tags_from_anidb_url, context)
-        self.assertEqual(response.status_code, 302)
+        self.assertEqual(response.status_code, 200)
```

Comment:
My bad ! I switched from `302` to `200` since this test failed and thought it came from modifications in my code. But looking back, the return code does not change.

Now, I found out the real cause of this test failing : I didn't update the `context` for the test, and especially `post` was modified to `confirm` 😅
This is now fixed !